--- aqfinance-0.9.108beta/src/lib/engine/lib/book/ae_query_stm.c.orig	2011-03-19 23:36:10.000000000 +0100
+++ aqfinance-0.9.108beta/src/lib/engine/lib/book/ae_query_stm.c	2017-02-11 16:24:05.011083592 +0100
@@ -143,11 +143,11 @@
   break
 
 #define CPYAMT(macro_fn)                           \
-  expr->valueData=AE_Statement_Get##macro_fn(st); \
+  expr->typedPtr.value=AE_Statement_Get##macro_fn(st); \
   break
 
 #define CPYDATE(macro_fn)                          \
-  expr->timeData=AE_Statement_Get##macro_fn(st);  \
+  expr->typedPtr.date=AE_Statement_Get##macro_fn(st);  \
   break
 
 
